src/main/java/duke.txt

```diff
@@ -1 +1,2 @@
-[E][ ] your father (at: Sep 9 2018)
+[T][ ] your mother
```

Comment:
Maybe can add the save file into gitIgnore! If not it'll keep popping up in your commits unless you wanna save your changes